src/helm/proxy/clients/palmyra_client.py

```diff
@@ -10,15 +16,45 @@
     DecodeRequestResult,
     TokenizationRequest,
     TokenizationRequestResult,
+    TokenizationToken,
 )
+from helm.common.general import ensure_file_downloaded
 from .client import Client, wrap_request_time, truncate_sequence
 
 
+_TOKENIZER_MODEL_URL = "https://huggingface.co/spaces/Writer/token-counter/resolve/92389981e4430f83383110728d954bda0f89fb30/tokenizer.model"  # noqa
+
+
+# SentencePieceProcessor is lazily initialized
+_sentence_piece_processor: Optional[SentencePieceProcessor] = None
+_sentence_piece_processor_directory: Optional[TemporaryDirectory] = None
+_sentence_piece_processor_lock = Lock()
```

Comment:
Why does this have to be a module-level variable as opposed to be in the class?

Reply:
I'd like to download this file once per process, rather than once per class instance (there can be many instances of the class).